pkg/config/load.go

```diff
@@ -31,6 +31,12 @@ type Prowgen struct {
 	// are private.
 	// This field has no effect if private is not set.
 	Expose bool `json:"expose,omitempty"`
+	// DisableAllRehearsals indicates that all jobs will not have their "can-be-rehearsed" label set
+	// and therefore will not be picked up for rehearsals.
+	DisableAllRehearsals bool `json:"disable_all_rehearsals,omitempty"`
```

Comment:
I don't mind the idea, but I think something like the following make more sense:
```
rehearsals:
  disable_all: false
  disabled_rehearsals:
  - test-1
  - test-2
  - test-3
```
If `disable_all` is true we would ignore the individual ones, otherwise only those would be disabled. I don't want people to have to call out the ones they want enabled.

Reply:
Nice! That makes more sense.